### ISSUE TITLE
force videojs-hls on apple browsers/devices

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-desktop#5323]
Sean has the details on how to force. Need to make sure it doesn't break the video display when scrolling.

**Comment by DispatchCommit** [comment]
https://github.com/lbryio/lbry-desktop/blob/21cbb64001ade24cf5d09b95dec3251d0dd7f2e5/ui/component/viewers/videoViewer/internal/videojs.jsx#L74

Default behavior is to *not* override in safari (PR#843 video js: https://github.com/videojs/http-streaming/pull/843/files#diff-51ba21f30e4636f6d930df4888344453235c45c1c9f1e3f0adc2bc6bdfa5ccb4R879)

There's no need to *always* force a native override as Safari supports HLS streaming natively, bitwave uses:
```
overrideNative: !videojs.browser.IS_SAFARI
```
Though `IS_ANY_SAFARI` is recommended (and default behavior now). 

The current usage of forcing to `true` is from an outdated recommendation here: https://github.com/videojs/http-streaming/issues/168

which stemmed from this PR: https://github.com/videojs/video.js/pull/5328

The reasoning for which to use is explained here: https://github.com/videojs/http-streaming/issues/505

> value should probably be: `overrideNative: !videojs.browser.IS_ANY_SAFARI`
> since iOS/iPadOS devices have good HLS support (just like macOS). Using `IS_SAFARI` would make the exception only cover desktop.
> 
>> we're likely going to use `IS_ANY_SAFARI` rather than IS_SAFARI as it covers more bases

### TLDR:The value for `overrideNative` should be changed from  to `!videojs.browser.IS_ANY_SAFARI`
